test/built-ins/object/values/function-property-descriptor.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) 2015 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Object.values should be writable, non-enumerable, and configurable
+es7id: pending
+author: Jordan Harband
+includes: [propertyHelper.js]
+---*/
+
+verifyConfigurable(Object, 'values');
```

Comment:
Ouch - if possible I'd prefer to just wait for that to be merged rather than adding a brittle workaround to test pollution